journal.py

```diff
@@ -30,11 +47,15 @@ def init_db():
     engine = sa.create_engine(DATABASE_URL)
     Base.metadata.create_all(engine)
 
+# @view_config(route_name='home', renderer='string')
+# def home(request):
+#     return "Hello World"
```

Comment:
No need to keep this here when pulling back into master. Your commit history will keep it around in case you need to go back to it. Try to keep your code clean.